<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceConfig</class>
 <widget class="QWidget" name="AppearanceConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tab">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tabPage1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <attribute name="title">
       <string>Icons</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>411</width>
         <height>350</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>ShutDown</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Reboot</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="KIconButton" name="kiconbutton">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="KIconButton" name="kiconbutton_2">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="KIconButton" name="kiconbutton_5">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>Hibernate</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="KIconButton" name="kiconbutton_4">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Log-Out</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="KIconButton" name="kiconbutton_7">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="checkBox_9">
          <property name="text">
           <string>Lock screen</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="KIconButton" name="kiconbutton_3">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>Suspend</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="KIconButton" name="kiconbutton_8">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QCheckBox" name="checkBox_11">
          <property name="text">
           <string>Switch user</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="KIconButton" name="kiconbutton_6">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QCheckBox" name="checkBox_8">
          <property name="text">
           <string>Turn-off screen</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabPage2">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <widget class="QComboBox" name="comboBoxBackground">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>210</y>
         <width>120</width>
         <height>25</height>
        </rect>
       </property>
       <property name="locale">
        <locale language="Italian" country="Italy"/>
       </property>
       <property name="maxVisibleItems">
        <number>4</number>
       </property>
       <item>
        <property name="text">
         <string>Default</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Standard</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Translucent</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>81</width>
         <height>70</height>
        </rect>
       </property>
       <property name="text">
        <string>Background</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxAskConfirmation">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>281</width>
         <height>23</height>
        </rect>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Ask for confirmation</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxLockBeforeSuspend">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>311</width>
         <height>23</height>
        </rect>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Lock screen for Hibernate and Suspend</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioOneRow">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>103</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1 row</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioTwoRows">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>103</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>2 rows</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioThreeRows">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>103</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>3 rows</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of rows:</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>kicondialog.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
